invenio_files_rest/__init__.py

```diff
@@ -6,7 +6,730 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Files download/upload REST API similar to S3 for Invenio."""
+
+r"""REST API for Files similar to S3 for Invenio.
+
+Invenio-Files-REST is a core component of Invenio which provides configurable
+REST APIs for uploading, serving, downloading and deleting files. It works in
```

Comment:
```
It works as a standalone module or in combination with `Invenio-Records <https://invenio-records.readthedocs.io>`_
through the `Invenio-Records-Files <https://invenio-records-files.readthedocs.io>`_ integration.
```

What do you think? And it would link to this https://github.com/inveniosoftware/invenio-records-files/pull/79/files#diff-f2377f695b4ac4cabc53efd87274b35eR184.

Feel free to rephrase if you like it.